src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,7 @@
+package course_project;
+
+public class Main {
+    public static void main(String[] args) {
+        new GameLauncher().launch();
```

Comment:
It's pretty uncomfortable to play... especially to put ships. Also, what rules did you use? I strongly remember there should be 1 4-cells ship and 4 1-cell ships...

Reply:
CARRIER(5),
    BATTLESHIP(4),
    CRUISER(3),
    SUBMARINE(3),
    DESTROYER(2);
    
    From your code

Reply:
Rules: https://www.hasbro.com/common/instruct/Battleship.PDF

Also, we were free to choose ui, as I remember

Reply:
In case of wrong input app shuts down... 
![image](https://user-images.githubusercontent.com/58872509/134550682-ffc8a242-814e-498d-9d40-87eff8c9485e.png)
Hitting the same cell is allowed

Reply:
![image](https://user-images.githubusercontent.com/58872509/134550842-0b322f87-9f75-4127-806a-019c6d39cf94.png)

Reply:
Interface is nice though, but the app is buggy. I'm sorry I can't accept that